Installation/changes.html

```diff
@@ -338,9 +338,11 @@ <h2 id="release4.10">Release 4.10 </h2>
     <li>New functions to read from and write to LAS/LAZ files (LIDAR
     format), with or without taking additional properties into
     account.</li>
-    <li>The API of the PLY function to read points with properties is
-    modified for unification with LAS. A new function to write PLY
-    with properties is provided.</li>
+    <li><b>Breaking change:</b> The API of the PLY function to read
+    points with properties is modified for unification with LAS
+    (see <code>CGAL::read_ply_points_with_properties()</code>). A new
+    function to write PLY with properties is provided
+    (<code>CGAL::write_las_points_with_properties()</code>).</li>
```

Comment:
I think you have swapped the two names of functions <code>CGAL::read_ply_points_with_properties()</code>/<code>CGAL::write_las_points_with_properties()</code> in the description.